internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,72 @@
 package daysteps
 
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	personaldata.Personal
+
+	Steps    int
+	Duration time.Duration
 }
 
-func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+func (ds *DaySteps) Parse(datastring string) error {
+	var (
+		steps    int
+		values   []string
+		duration time.Duration
+		err      error
+	)
+
+	values = strings.Split(datastring, ",")
+	if len(values) != 2 {
+		return fmt.Errorf("two comma separated values are expected, %q given", datastring)
+	}
+
+	steps, err = strconv.Atoi(values[0])
+	if err != nil {
+		return fmt.Errorf("cannot parse first value (steps): %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("cannot parse first value (steps): %w", err)
```

Reply:
Нужно везде поправить